arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1550,7 +1575,7 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition."
```

Comment:
@anuchandy, the build has failed in $MODE == 'node', can you help me resolving it.